packages/plugins/graphql/server/src/services/builders/dynamic-zones.ts

```diff
@@ -88,7 +88,11 @@ export default ({ strapi }: Context) => {
      * @param {string} inputName - the name of the dynamic zone's input
      * @return {[NexusUnionTypeDef, NexusScalarTypeDef]}
      */
-    buildDynamicZoneDefinition(definition: Attribute.DynamicZone, name: string, inputName: string) {
+    buildDynamicZoneDefinition(
```

Comment:
Since his is in TS and typed already, do we need the jsdoc?

Reply:
We don't